organization/_base.js

```diff
@@ -37,6 +37,8 @@ module.exports = {
     'arrow-spacing': 2,
     // verify super() callings in constructors
     'constructor-super': 2,
+    // enforce require() on the top-level module scope
+    'global-require': 1,
```

Comment:
It was added in `eslint-config-airbnb@8.0.0`, however, I'm not sure that it must throw error. So, I've overrided this rule with warning.